preprod.gen3.biodatacatalyst.nhlbi.nih.gov/portal/gitops.json

```diff
@@ -707,6 +719,12 @@
       {
         "name": "dbGaP Accession Number",
         "field": "study_id"
+      },
+      {
+        "name": "Released",
+        "field": "release_state",
```

Comment:
if we're using Yes, No, Partially, Obsolete, we should change this field to be `released` too